### ISSUE TITLE
Indicate required fields rather than optional 

### ISSUE THREAD

**Issue by j6design** [sfbrigade/adopt-a-drain#143]

**Comment by jeanwalshie** [comment]
@j6design I like that idea, too!

**Comment by jszwedko** [comment]
@j6design I think it was an intentional design decision to mark optional fields rather than required. cc/ @jasonlally

**Comment by jeanwalshie** [comment]
i agree that it's looking a little busy/complicated to say: (visible to others, optional) (visible to others) (private) 

i would vote for using the little red star (or another universal symbol) to denote mandatory fields and keeping the "private" vs. "visible to others" distinction.

**Comment by jszwedko** [comment]
@jasonlally mentioned before that users are more likely to fill in if optional fields were marked rather than required. I don't have a strong opinion either way, it is an easy change, but it would be good to get consensus.

**Comment by jasonlally** [comment]
There were a mix of considerations here. In the original app, it was even busier and there were lots of icons that created some cognitive load to decipher. I think a lock indicated private, but not necessarily required. It was a little confusing.

Anyway, I think the intention of the original developers was to be transparent that certain information would be made available to other users. We could likely simplify, drop mobile phone from the initial sign up (since we don't really use it).

Then just mark things (optional).

Here's a justification as to the various reasons to mark fields optional as opposed to required and to avoid using the red asterisk: https://www.formulate.com.au/blog/required-versus-optional-fields-new-standard

It's certainly not scripture, but I found it a lot easier to parse the form visually quickly than with icons or asterisks.

I'm open to additional approaches, but I think we could just:

1. Drop mobile from the initial sign up
2. Change all optional fields to just say optional
3. Add a short note at the top that says: "All fields required, unless marked optional. Note: all fields, except password, are visible to other users."

**Comment by j6design** [comment]
Although it would be great to conduct a little A/B testing, I agree with trying Jason's proposed approach.  

Note: If possible, could we move away from asking optional information on signup forms,
